package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "openfisca.org",
+  "version": "1.0.0",
+  "description": "The public-facing website of OpenFisca for an international audience.",
+  "devDependencies": {
+    "eslint": "^4.4.1",
+    "eslint-config-google": "^0.9.1",
+    "gh-pages": "^1.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openfisca/openfisca.org.git"
+  },
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/openfisca/openfisca.org/issues"
+  },
+  "homepage": "https://github.com/openfisca/openfisca.org#readme",
+  "scripts": {
+    "test": "./node_modules/.bin/eslint publish.js",
```

Comment:
> The good thing with NPM scripts is that NPM adds the `.bin` folder automatically to the `PATH`. That means you can just use `eslint` as if it was installed globally, rather than specifying the local path 🙂